src/nfs/nfs_server_impl.cpp

```diff
@@ -42,10 +42,11 @@
 namespace dsn {
 namespace service {
 
-DSN_DEFINE_int32("nfs",
-                 max_send_rate_megabytes_per_disk,
-                 500,
-                 "max rate per disk of send to remote node(MB/s)");
+DSN_DEFINE_uint32(
+    "nfs",
+    max_send_rate_megabytes_per_disk,
```

Comment:
I notice that this pull request adds `group_validator` for `max_copy_rate_megabytes_per_disk`, but `max_send_rate_megabytes_per_disk` not. Should max_send_rate_megabytes_per_disk also add the validator?

Reply:
send side don't have `FLAGS_nfs_copy_block_bytes` variable,  copy side validating is enough